seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3844,6 +3844,7 @@
         "assert(switchOfStuff(\"d\") === \"stuff\", 'message: <code>switchOfStuff(\"d\")</code> should have a value of \"stuff\"');",
         "assert(switchOfStuff(4) === \"stuff\", 'message: <code>switchOfStuff(4)</code> should have a value of \"stuff\"');",
         "assert(!/else/g.test(code) || !/if/g.test(code), 'message: You should not use any <code>if</code> or <code>else</code> statements');",
+        "assert(switchOfStuff(19372) === \"stuff\", 'message: You should use a <code>default</code> statement');",
```

Comment:
Just for clarity, can you change this line to "assert(switchOfStuff("string-to-trigger-default-case") ...

Just so that people coming behind us know exactly what that test is doing with just a quick glance.

Many thanks :+1: